src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ CHANGELOG
  * Add a `CssColor` constraint to validate CSS colors
  * Add support for `ConstraintViolationList::createFromMessage()`
  * Add error's uid to `Count` and `Length` constraints with "exactly" option enabled
+ * Allow `Sequence` constraint to be applied onto class as an attribute
```

Comment:
CHANGELOG files are modified for new features only. You can safely drop this line.
This bug fix will appear on the release changelog on github and the Symfony website.

Reply:
Exactly. Please remove the changelog entry.

Reply:
done. thankx